assets/js/index.js

```diff
@@ -0,0 +1,161 @@
+const workingHours = [
+  {
+    label: "9AM",
+    key: 9,
+  },
+  {
+    label: "10AM",
+    key: 10,
+  },
+  {
+    label: "11AM",
+    key: 11,
+  },
+  {
+    label: "12AM",
+    key: 12,
+  },
+  {
+    label: "1PM",
+    key: 13,
+  },
+  {
+    label: "2PM",
+    key: 14,
+  },
+  {
+    label: "3PM",
+    key: 15,
+  },
+  {
+    label: "4PM",
+    key: 16,
+  },
+  {
+    label: "5PM",
+    key: 17,
+  },
+];
+const timeBlocks = $("#time-blocks");
+//accessing date and time
+const currentDateStamp = moment();
+// add
+const renderDate = () => {
+  return moment().format("dddd, MMMM Do, YYYY");
+};
+
+const renderTime = () => {
+  return moment().format("h:mm:ss a");
+};
+
+// target save button
+//const saveButton = $("#saveBtn");
+//saveButton.on("click", function () {
+// console.log("button clicked");
+//});
+
+$("#currentDay").text(renderDate);
+$("#currentTime").text(renderTime);
+
+const getEventForTimeBlock = (hour) => {
+  const tasks = readFromLocalStorage("planner", workingHours);
+  // get task by hour from LS
+  console.log(tasks);
+  let task = tasks[hour];
+  if (task == undefined) {
+    task = "";
+    return task;
+  }
+  return task;
+};
```

Comment:
Make sure you leave a spare line in between each function so the code is clearer and easier to read.